New 5.0 Characters and Missing Collei Info

keihachimaru opened the pull request:
Addition of information about Kachina, Mualani, and Kinich in english. Kinich information is not yet complete, because he has not been released yet.
Edit: Issues #300 and #310.

keihachimaru committed:
New 5.0 Characters

Addition of information about Kachina, Mualani, and Kinich in english.

keihachimaru committed:
Missing info

Missing name and description for Collei's Normal Attack

niktekusho commented:
Fixes #310 😉

man90es reviewed (changes requested):
Please add Kinich data from this page: https://gensh.honeyhunterworld.com/kinich_101/?lang=EN

His release date is 18 September

keihachimaru committed:
Missing Kinich Data and Mualani Talents Upgrades

keihachimaru committed:
Merge branch 'mistress' into mistress

keihachimaru commented:
Ok, I've added them. Also I added Mualani's Talents upgrades info.

man90es commented on assets/data/characters/kinich/en.json:
```diff
@@ -0,0 +1,303 @@
+    "nation": "Natlan",
+    "affiliation": "Scions of the Canopy",
+    "rarity": 5,
+    "release": "",
```
18 September 2024

man90es commented on assets/data/characters/kinich/en.json:
```diff
@@ -0,0 +1,303 @@
+    "rarity": 5,
+    "release": "",
+    "constellation": "Chimaera Alebriius",
+    "birthday": "",
```
11 November

keihachimaru committed:
Update en.json

keihachimaru committed:
Merge branch 'mistress' of https://github.com/keihachimaru/genshin.dev-api into mistress

M4GICB reviewed (changes requested):
It would also be good to get these new characters added to the talent material lists

keihachimaru committed:
Material Information

- Boss Materials
- Character Ascension Materials
- Common Ascension Materials
- Local Specialties
- Talent Books
- Talent Bosses

man90es commented:
@M4GICB I didn't look at the latest commit, leaving it to you since you requested it. The rest looks good to me
